src/qt/locale/bitcoin_it.ts

```diff
@@ -4310,187 +4310,187 @@ p, li { white-space: pre-wrap; }
-        <translation type="unfinished"></translation>
+        <translation>Aggiungi i peers che si connettono da indirizzi IP predefiniti (e.g. 1.2.3.4) o da reti specificate in notazione CIDR (e.g. 1.2.3.0/24).</translation>
     </message>
     <message>
         <source>%s is set very high!</source>
-        <translation type="unfinished"></translation>
+        <translation>Il valore %s é molto alto!</translation>
     </message>
     <message>
         <source>Amaze</source>
-        <translation type="unfinished"></translation>
+        <translation>Fantastico</translation>
     </message>
     <message>
         <source>Chain selection options:</source>
-        <translation type="unfinished"></translation>
+        <translation>Seleziona una chain:</translation>
     </message>
     <message>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation>Default</translation>
     </message>
     <message>
         <source>Keypool ran out, please call keypoolrefill first</source>
-        <translation type="unfinished"></translation>
+        <translation>La keypool é vuota, usa keypoolrefill prima di continuare</translation>
     </message>
     <message>
         <source>Many generous</source>
-        <translation type="unfinished"></translation>
+        <translation>Molto generoso</translation>
     </message>
     <message>
         <source>Minimum</source>
-        <translation type="unfinished"></translation>
+        <translation>Minimo</translation>
     </message>
     <message>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation>Continua</translation>
     </message>
     <message>
         <source>Receive and display P2P network alerts (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricevi e mostra avvisi relativi alla reta P2P (default: %u)</translation>
     </message>
     <message>
         <source>Send transactions with full-RBF opt-in enabled (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>Invia transazioni che supportano l&apos;opzione di sostituzione con commissione (RBF) (default: %u)</translation>
     </message>
     <message>
         <source>Starting network threads...</source>
-        <translation type="unfinished"></translation>
+        <translation>Inizializzazione dei thread di rete...</translation>
     </message>
     <message>
         <source>Such expensive</source>
-        <translation type="unfinished"></translation>
+        <translation>Costosissimo!</translation>
     </message>
     <message>
         <source>The wallet will avoid paying less than the minimum relay fee.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il portafoglio preverrá il pagamento di commissioni inferiori alle commissioni minime</translation>
```

Comment:
<translation>Il portafoglio eviterà il pagamento di commissioni inferiori alle commissioni minime</translation>

Reply:
Hmmm maybe. I am bit on the fence on this one but we can go for your version.